rtsp_test.go

```diff
@@ -74,11 +74,10 @@ func TestRTSPCamera(t *testing.T) {
 			defer imageTimeoutCancel()
 			var im image.Image
 			for imageTimeoutCtx.Err() == nil {
-				img, f, err := camera.ReadImage(imageTimeoutCtx, rtspCam)
+				img, err := camera.DecodeImageFromCamera(imageTimeoutCtx, "image/jpeg", nil, rtspCam)
```

Comment:
Could we import the mimetype enum from RDK here?

Reply:
yeah should use enum lookup in tests here -- will fix.